Dockerfile

```diff
@@ -1,13 +1,7 @@
-FROM renovate/ubuntu:18.04@sha256:f94cb492cb94e913186e477168dff87ab4c12c08a445d65d511c0cc628369d69
+FROM renovate/buildpack
 
-USER root
+ENV JAVA_VERSION=8
```

Comment:
I had expected it to be `ARG`. Which takes precedence if it's also set in environment when run?

Reply:
When using `ARG` it will not exist on runtime, so i could do

```Dockerfile
ARG JAVA_VERSION=8
ENV JAVA_VERSION=${JAVA_VERSION}

Reply:
I wasn't aware of that distinction. I just figured we may want the ability to supply the value from CLI